src/main/java/bio/terra/janitor/service/cleanup/FlightSubmissionFactoryImpl.java

```diff
@@ -32,4 +40,16 @@ public UnsupportedCleanupFlight(FlightMap inputParameters, Object applicationCon
       addStep(new FinalCleanupStep(janitorDao));
     }
   }
+
+  /** A Flight to cleanup a GoogleBucket. */
+  public static class GoogleBucketCleanupFlight extends Flight {
+    public GoogleBucketCleanupFlight(FlightMap inputParameters, Object applicationContext) {
+      super(inputParameters, applicationContext);
+      JanitorDao janitorDao =
+          ((ApplicationContext) applicationContext).getBean("janitorDao", JanitorDao.class);
+      addStep(new InitialCleanupStep(janitorDao));
+      addStep(new GoogleBucketCleanupStep());
```

Comment:
we should add some retries to the cleanup step.

Reply:
The benefit of using CRL is it already handles retries for us. Do we want to add more retry logic on top of that?